client/signup/steps/domains/style.scss

```diff
@@ -123,11 +123,33 @@ body.is-section-signup.is-white-signup {
 			}
 		}
 
+		.register-domain-step .domains__domain-side-content-container-mobile-experiment {
+			display: flex;
+			height: 100%;
```

Comment:
There's a large gap below the browser item that looks a bit odd to me. What do you think about reducing it? 

I tested by removing the `height: 100%` and added a `margin-bottom: 20px` to `.domains__domain-side-content-container-browser-chrome`

Thoughts?

<img width="427" alt="Screen Shot 2022-03-04 at 5 15 47 PM" src="https://user-images.githubusercontent.com/793774/156849301-1efb68d7-3180-4c17-aeb2-c865efb982c4.png">

Reply:
> Thoughts?

In the designs that @annchichi created, she has the "Already own a domain" at the very bottom of the page. Which I thought looked nice. So I tried to replicate this.